cyw43/src/ble_connector.rs

```diff
@@ -0,0 +1,64 @@
+use embedded_hal_1::digital::OutputPin;
+
+use crate::{Runner, SpiBusCyw43};
+
+pub struct BleConnector<'a, PWR, SPI> {
+    runner: Runner<'a, PWR, SPI>,
+}
+
+impl<'a, PWR, SPI> BleConnector<'a, PWR, SPI>
+where
+    PWR: OutputPin,
+    SPI: SpiBusCyw43,
+{
+    pub fn new(runner: Runner<'a, PWR, SPI>) -> BleConnector<'a, PWR, SPI> {
+        return BleConnector { runner };
```

Comment:
@Dirbaio sorry for the ping but I could use some guidance. I think you know as well as I do we can't use `runner` here due to borrowing checking. I know before you mentioned maybe putting `Bus` behind a mutex but I didn't really have great success with that either.

How would you recommend an `HciConnector` makes its way back to `runner.bus` to be able to do `hci_read()` and `hci_write()`? A channel? Waker like ioctl does?

Same for refactoring the bluetooth firmware init, it "has to happen" after CLM load.